aea/protocols/generator/base.py

```diff
@@ -1521,6 +1527,8 @@ def _serialization_class_str(self) -> str:
             )
         )
 
+        cls_str += "# pylint: disable=no-member\n"
```

Comment:
if added to each line, it does not work well with black

Reply:
![image](https://user-images.githubusercontent.com/9467617/112724444-b3492d00-8f13-11eb-85e0-0472e54281ab.png)